### PR TITLE
[v2 rc] storagePrefix

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

File: packages/client/test/client.spec.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { delay } from "@walletconnect/time";
```
Comment (bkrem) [inline packages/client/test/client.spec.ts]: Accidental import?

File: packages/types/src/store.ts [hunk 2]
```diff
@@ -13,7 +13,13 @@ export abstract class IStore<Key, Value> {
 
   public abstract readonly values: Value[];
 
-  constructor(public core: ICore, public logger: Logger, public name: string) {}
+  constructor(
+    public core: ICore,
+    public logger: Logger,
+    public name: string,
+    // @ts-ignore
```
Comment (bkrem) [inline packages/types/src/store.ts:20]: Is the `@ts-ignore` still needed?

**Review by bkrem — APPROVED** [review]

**Review by xzilja — APPROVED** [review]
Looks good, but can we not just

```ts
  get storageKey(): string {
    return CLIENT_STORAGE_PREFIX + this.version + "//" + this.name;
  }
```

And thus get rid of `private storagePrefix` in all classes? As well as code that assigns it

File: packages/client/test/client.spec.ts (hunk 1 above)
Comment (pedrouid) [reply packages/client/test/client.spec.ts]: Yes, I was trying to fix ping test

File: packages/types/src/store.ts (hunk 2 above)
Comment (pedrouid) [reply packages/types/src/store.ts:20]: It wouldn’t compile without it

**Comment by pedrouid** [comment]
> Looks good, but can we not just
> 
> ```ts
>   get storageKey(): string {
>     return CLIENT_STORAGE_PREFIX + this.version + "//" + this.name;
>   }
> ```
> 
> And thus get rid of `private storagePrefix` in all classes? As well as code that assigns it

True, but made it consistent with store.ts which needs it

File: packages/client/test/client.spec.ts (hunk 1 above)
Comment (bkrem) [inline packages/client/test/client.spec.ts]:
```suggestion

```

**Pull request merged**